docs/man-pages/condor_token_request.rst

```diff
@@ -0,0 +1,136 @@
+
+
+*condor_token_request*
+======================
+
+request a token from a remote daemon
+:index:`condor_token_request<single: condor_token_request; HTCondor commands>`\ :index:`condor_token_request command`
+
+Synopsis
+--------
+
+**condor_token_request** [**-identity** *user@domain*] [**-authz** *authz* ...]
+[**-lifetime** *value*]
+[**-pool** *pool_name*] [**-name** hostname] [**-type** *type*]
+[**-token** *filename*]
+
+**condor_token_request** [**-help** ]
+
+Description
+-----------
+
+*condor_token_request* will request an authentication token from a remote
+daemon.  Unlike *condor_token_fetch*, the user doesn't need to have an existing
```

Comment:
Swap the "Unlike..." and "Token requests..." sentences; IMO the more important part of the description is that this command needs manual action from someone else.